<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeSkyboxDialog</class>
 <widget class="QDialog" name="ChangeSkyboxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>changeskyboxDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>A single picture</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6 pictures</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="skyboxImageImporter">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="singlePictureWidget">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Picture</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="picturePathText">
         <property name="placeholderText">
          <string>Picture Path</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="importSingleButton">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MultiplePicturesWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Top</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textTop">
            <property name="placeholderText">
             <string>File path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importTopButton">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Down</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textBottom">
            <property name="placeholderText">
             <string>File path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importDownButton">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textLeft">
            <property name="placeholderText">
             <string>File path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importLeftButton">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Right</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textRight">
            <property name="placeholderText">
             <string>File path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importRightButton">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Front</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textFront">
            <property name="placeholderText">
             <string>File path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importFrontButton">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="textBack">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>File path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="importBackButton">
            <property name="text">
             <string>Import</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="modifyButton">
        <property name="text">
         <string>Modify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>skyboxImageImporter</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
